fili-sql/src/main/java/com/yahoo/bard/webservice/sql/SqlAggregationQuery.java

```diff
@@ -16,11 +16,11 @@
 import com.yahoo.bard.webservice.druid.model.query.DruidFactQuery;
 import com.yahoo.bard.webservice.data.time.Granularity;
 import com.yahoo.bard.webservice.druid.model.query.QueryContext;
+import com.yahoo.bard.webservice.druid.model.virtualcolumns.VirtualColumn;
 
 import org.joda.time.Interval;
 
-import java.util.Collection;
-import java.util.Optional;
+import java.util.*;
```

Comment:
your IDE is globbing imports when it shouldn't.  There's a setting for this in IDEA assuming that's what you're using.